src/shared_utilities/drivers.jl

```diff
@@ -405,7 +431,7 @@ from unit conversion of evaporation from a mass to a liquid water volume
 flux. When r_sfc is nonzero, an additional resistance is applied in series
 to the vapor flux (and hence also the latent heat flux).
 """
-function turbulent_fluxes_at_a_point(
+function turbulent_fluxes_helper(
```

Comment:
Do you think it would be clearer to call this "compute_turbulent_fluxes_at_a_point"?

It's a long name, but we have a number of "turbulent fluxes" functions now, just wondering if we could find a more descriptive name than "helper"

Reply:
or maybe "full_turbulent_fluxes_at_a_point" (because it returns momentum fluxes too? not sure)

Reply:
or maybe - the outer function should be the helper? and the inner one "turbulent_fluxes_at_a_point"? not sure!

Reply:
Yes good point! I wanted to keep the outer one as `turbulent_fluxes_at_a_point` to minimize the changes we have to make. I'll rename the helper to `compute_turbulent_fluxes_at_a_point`, if that's okay with you